Friday Fixes

alexvandesande opened the pull request:
* Fixes the BigNumber issue on Metamask
* Defaults extra deposit amount to 0
* Adds a gas price picker, from 40 shannon to 1.25 (with an alert that this might not go through)
* Hides the extra deposit and gas picker into a “more options” link
* Adds alert that you can only bid once
* More tolerant of non-ascii characters
* Removes short names and capitalized names from the chaff hash choices

Commit:
advanced fee mode

Commit:
rebuild

Arachnid commented on app/imports/templates/status/open.js:
```diff
@@ -9,14 +9,17 @@ Template['status-open'].events({
     } else {
       TemplateVar.set(template, 'opening-' + name, true)
-      registrar.openAuction(name, TemplateVar.getFrom('.new-bid', 'dummyHashes'), {
+      registrar.openAuction(name, hashes, {
```
Can we do a check to make sure that `hashes` isn't empty here?

alexvandesande replied:
Made further checks.

Commit:
Makes wording around the character limit more clear.

As a user, when I went to the site and found that my last name was unavailable because it was too short I was quite miffed.  I didn't find the wording to be clear enough that the limitations were temporary nor did I know where to go for more details.  This change just adds some emphasis, makes it painfully clear that the limit will eventually be reduced, and provides a link to more details.

Commit:
more checks that the dictionary is loaded

Commit:
adds note saying char limit is temp

Commit:
merge

Commit:
merge master
